providers/mandrill/src/lib/mandrill.provider.ts

```diff
@@ -5,11 +5,25 @@ import {
   ISendMessageSuccessResponse,
   ICheckIntegrationResponse,
   CheckIntegrationResponseEnum,
+  IEmailEventBody,
+  EmailEventStatusEnum,
 } from '@novu/stateless';
 
 import mailchimp from '@mailchimp/mailchimp_transactional';
 import { MandrillInterface } from './mandrill.interface';
 
+export enum MandrillStatusEnum {
+  OPEN = 'opened',
+  SEND = 'sent',
+  DEFERRAL = 'deferred',
+  HARD_BOUNCE = 'hard_bounced',
+  SOFT_BOUNCE = 'soft_bounced',
+  CLICK = 'clicked',
+  SPAM = 'spam',
+  UNSUB = 'unsubscribed',
+  REJECT = 'rejected',
+}
```

Comment:
You should keep the status naming in participles, for a meaning thing as they are status not actions.
```suggestion
export enum MandrillStatusEnum {
  OPENED = 'opened',
  SENT = 'sent',
  SOFT_BOUNCED = 'soft_bounced',
  CLICKED = 'clicked',
  REJECTED = 'rejected',
  ...
}
```